graphics/macgui/macfontmanager.cpp

```diff
@@ -516,6 +516,11 @@ const Font *MacFontManager::getFont(MacFont *macFont) {
 				font = Graphics::loadTTFFontFromArchive("FreeSans.ttf", macFont->getSize(), Graphics::kTTFSizeModeCharacter, 0, 0, Graphics::kTTFRenderModeMonochrome);
 				_uniFonts[macFont->getSize()] = font;
 			}
+		} else {
```

Comment:
It is not clear to me why it is in the non-Unicode branch. Also, what does happen if there is no font registered? I see no return value checking.

Reply:
Why did you just mark it without any changes or address the issue?

Reply:
When registering the font family, I do check the return value. As for why this code is in the non-Unicode branch, I believe it’s to avoid altering the original logic of `getFont`. Moreover, if I place the code inside the `kWMModeUnicode` branch, it seems not to run properly.

Reply:
Please explain "not to run properly"

Reply:
What I mean is, that it seems like the program didn’t even enter the above branch at all.
![image](https://github.com/scummvm/scummvm/assets/101713235/c0cde485-72c5-42dc-9e35-96c83e7b6d48)
![image](https://github.com/scummvm/scummvm/assets/101713235/b3604867-de2d-4a06-8d55-3a1362c82b33)

Reply:
So, make it run in the Unicode mode?

Reply:
ok, I'll try